GameServerCore/Packets/Handlers/IPacketHandler.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace GameServerCore.Packets.Handlers
+{
+    public interface IPacketHandler
```

Comment:
Posibbly move this to packet definitions?

Reply:
Currently lib use one of the implementors of that interface (PacketDefinitions get all the handlers from Game class, and send each packet to the corresponding handler).

Ill change this mechanism in another PR (it is one of the lines in my #732 issue).